libvast/vast/system/sink.hpp

```diff
@@ -80,26 +83,35 @@ caf::behavior sink(caf::stateful_actor<sink_state<Writer>>* self,
     }
   );
   return {
-    [=](const std::vector<event>& xs) {
+    [=](std::vector<event>& xs) {
```

Comment:
Why did you drop constness here? Is this just temporary until writers will work with table slices? If so, then I'm fine with this convenience change to resize the batch later.

Reply:
Yes, that's only a workaround for now. Eventually we'll switch to table slices.